tests/test_formparser.py

```diff
@@ -399,6 +399,24 @@ def test_empty_multipart(self):
         assert files == MultiDict()
 
 
+class TestMultiPartParser(object):
+
+    def test_constructor_not_pass_stream_factory_and_cls(self):
+        parser = formparser.MultiPartParser()
+
+        assert parser.stream_factory == formparser.default_stream_factory
```

Comment:
You are right, `is` is more stricter. But in that case I think == is enough because there is used check by ref.